soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -1477,6 +1483,26 @@ void Randomizer::LoadHintLocations(const char* spoilerFileName) {
         ParseHintLocationsFile(spoilerFileName);
     }
 
+    CustomMessageManager::Instance->ClearMessageTable(Randomizer::hintMessageTableID);
+    CustomMessageManager::Instance->AddCustomMessageTable(Randomizer::hintMessageTableID);
+
+    CustomMessageManager::Instance->CreateMessage(
+        Randomizer::hintMessageTableID, 0x7040,
+                                           { TEXTBOX_TYPE_BLUE, TEXTBOX_POS_BOTTOM, gSaveContext.childAltarText,
+                                             gSaveContext.childAltarText, gSaveContext.childAltarText });
+    CustomMessageManager::Instance->CreateMessage(
+        Randomizer::hintMessageTableID, 0x7088,
+                                           { TEXTBOX_TYPE_BLUE, TEXTBOX_POS_BOTTOM, gSaveContext.adultAltarText,
+                                             gSaveContext.adultAltarText, gSaveContext.adultAltarText });
+    CustomMessageManager::Instance->CreateMessage(
+        Randomizer::hintMessageTableID, 0x70CC,
+                                           { TEXTBOX_TYPE_BLACK, TEXTBOX_POS_BOTTOM, gSaveContext.ganonHintText,
+                                             gSaveContext.ganonHintText, gSaveContext.ganonHintText });
+    CustomMessageManager::Instance->CreateMessage(
+        Randomizer::hintMessageTableID, 0x70CD,
+                                           { TEXTBOX_TYPE_BLACK, TEXTBOX_POS_BOTTOM, gSaveContext.ganonText,
+                                             gSaveContext.ganonText, gSaveContext.ganonText });
```

Comment:
looks like you have the enum values for these in `CustomMessageTypes.h`, let's use them!

Reply:
I added them in there because I needed access to them in areas where `#include message_data_static.h` caused type redefinition errors. I suppose if that's all we need from there I can switch the include from `message_data_static.h` to `CustomMessageTypes.h`

Reply:
Oh wait I confused myself here. Yes I just missed the raw textIDs here.